docs/docs/configuration/server.rst

```diff
@@ -229,6 +229,50 @@ timeout
     specify a timeout in seconds for blocking operations like the
     connection attempt.
 
+Mail
+----
+
+Configure the format the mail notification uses in the SMTP section.
+
+.. code-block:: ini
+
+    [mail]
+    subject_admin = {title}
+    subject_user = Re: New comment posted on {title}
+
+subject_admin
+    specify the subject format of the notification email sent to the admin.
+    Default: ``{title}``.
+    Available variables:
+
+    {title}
+        The title of the isso thread.
+
+    {replier}
+        The name of the author of the comment. If the author has no name,
+        it will fallback to the term "Anonymous".
+
+subject_user
+    specify the subject format of the notification email sent to the subscribed
+    commenter. If it is set to two values, then the first one will be used when
```

Comment:
Why not just have two settings, one for new messages and one for replies?

Reply:
Okay, I'll seperate it into two options.